bundles/scripting/sightly/java-compiler/src/main/java/org/apache/sling/scripting/sightly/render/AbstractRuntimeObjectModel.java

```diff
@@ -67,6 +67,16 @@ public boolean isPrimitive(Object obj) {
     }
 
     @Override
+    public boolean isDate(Object target) {
+        return (target instanceof Date);
```

Comment:
Since the HTL specification will include explicit option for timezone (and Calendar is a Date + Timezone) I don't think it's really that helpful to provide shorthands for Calendar object formatting (their internal timezone might clash with the user-specified one). The HTL syntax makes it trivial to derive date and timezone from a Calendar: `${'yyyy-MM-dd HH:mm' @ format=myCal.time, timezone=myCal.timeZone}`